README.adoc

```diff
@@ -8,48 +8,76 @@ See https://www.youtube.com/watch?v=sOWC4qrXxFk[the video demo].
 
-=== Binary S2I: build locally and deploy to OpenShift Online
+=== Locally with OpenShift Origin
 
-Create the project, build the `war` file locally and provision it on OpenShift:
+. If you just installed oc client tools, follow the https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md[OpenShift Local Cluster Management] docs to get `oc cluster (up|down)` working.
+. `oc cluster up`
+. `oc login`, enter "developer" for both username and password when prompted
+. `oc new-project <your-project-name>` (ex: "openshift", must consist of only lowercase letters, numbers and hypens)
+. `oc new-app --image-stream="openshift/wildfly:10.1" https://github.com/kiegroup/optashift-employee-rostering.git#master`
 
-```
-./provision.sh setup employee-rostering --binary
-```
+To trigger a new build with the latest updates from the master branch, run
 
-That's it, it should now be running on the web.
-Go to https://manage.openshift.com/[the OpenShift Online web console]
-to get the `*.openshiftapps.com` URL.
+`oc start-build optashift-employee-rostering`
 
-Later, you can just run this command to deploy new code changes:
+To delete the OpenShift application and project, run
 
-```
-./provision.sh deploy employee-rostering --binary
-```
+`oc delete project <your-project-name>`
 
+=== OpenShift Dedicated
 
-=== Normal S2I: build from source on OpenShift Dedicated
+. Login to OpenShift Dedicated from command line
++
+(With username and password)
++
+`oc login -u=<username> -p=<password> --server=<your-openshift-server> --insecure-skip-tls-verify`
++
+(With OAuth token)
++
+`oc login <https://api.your-openshift-server.com> --token=<tokenID>`
 
-WARNING: This currently does not work on OpenShift Online due to resource limits.
+. `oc new-project <your-project-name>` (ex: "openshift", must consist of only lowercase letters, numbers and hypens)
+. `oc new-app --image-stream="openshift/wildfly:10.1" https://github.com/kiegroup/optashift-employee-rostering.git#master`
 
-Build it entirely from source on OpenShift and provision it there:
+To trigger a new build with the latest updates from the master branch, run
 
-```
-./provision.sh deploy employee-rostering
-```
+`oc start-build optashift-employee-rostering`
 
+To delete the OpenShift application and project, run
 
-=== Delete the OpenShift application and project
+`oc delete project <your-project-name>`
 
-```
-./provision.sh delete employee-rostering
-```
+=== OpenShift Online
+
+Login to OpenShift Online from command line
+
+(With username and password)
+
+`oc login -u=<username> -p=<password> --server=<your-openshift-server> --insecure-skip-tls-verify`
+
+(With OAuth token)
+
+`oc login <https://api.your-openshift-server.com> --token=<tokenID>`
+
+Create the project, build the `war` file locally and provision it on OpenShift:
+
+`./provision.sh setup employee-rostering --binary`
+
```

Comment:
Can we add something here (in a new PR, this is merged already) to motivate why the normal way doesn't work?
Something like:

NOTE
====
Creating a new app from the GitHub URL on OpenShift online doesn't work because the build pod's maven process fails with error code 137 due to lack of memory for the GWT compilation.
====